From d5c4a098d7447df782f0f23bd9b3ff59648fdded Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Tue, 28 Aug 2012 11:17:49 -0400
Subject: [PATCH] Call g_thread_init() conditionally

An explicit call to g_thread_init() is not needed with >=glib-2.24, and
causes build failure with >=glib-2.31. Since evolution-webcal is
theoretically compatible with >=glib-2.8, guard the call with #ifdef
instead of just removing it.
---
 src/evolution-webcal-main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/evolution-webcal-main.c b/src/evolution-webcal-main.c
index f675686..b6611d3 100644
--- a/src/evolution-webcal-main.c
+++ b/src/evolution-webcal-main.c
@@ -250,7 +250,9 @@ gint main (gint argc, gchar ** argv) {
   textdomain (GETTEXT_PACKAGE);
 #endif
 
+  #if !GLIB_CHECK_VERSION(2,31,0)
   g_thread_init (NULL);
+  #endif
 
   if (!gtk_init_with_args (&argc, &argv,
 			_("- Evolution webcal: URI Handler"),
-- 
1.7.12

